src/NuGet.Core/NuGet.Packaging/Signing/Native/SignedCmsWrapper .cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Packaging.Signing
+{
+#if IS_SIGNING_SUPPORTED && NETSTANDARD2_1
+    internal class SignedCmsWrapper : ICms
+    {
+        private SignedCms _signedCms;
+
+        public SignedCmsWrapper(SignedCms signedCms)
+        {
+            _signedCms = signedCms;
```

Comment:
According to CmsFactory, the nativeCms will only be null if the framework is not supported. Since I added throwing NotSupportedException in CmsFactory for unsupported framework, do I still need to do the null check? If so, what shall do if it's null? Thanks!

Reply:
How about adding argument checking to `CmsFactory` instead?

Reply:
Thanks! Added.